assets/src/components/parts/janus-navigation-button/NavigationButton.tsx

```diff
@@ -429,8 +448,28 @@ const NavigationButton: React.FC<PartComponentProps<NavButtonModel>> = (props) =
   };
 
   return ready && buttonVisible ? (
-    <button data-janus-type={tagName} {...buttonProps} style={styles}>
-      {buttonTitle}
+    <button
+      data-janus-type={tagName}
+      {...buttonProps}
+      style={{
+        ...styles,
+        display: 'flex',
```

Comment:
it's been a while, so I don't remember if this is even an option, but it seems like it would be nice to have these "built-in styles" to be not inline.